processor/groupbyattrsprocessor/README.md

```diff
@@ -4,13 +4,18 @@ Supported pipeline types: **traces**, **logs**, **metrics**
 
 ## Description
 
-This processor re-associates spans, log records and metric datapoints to a *Resource* that matches with the specified attributes. As a result, all spans, log records or metric datapoints with the same values for the specified attributes are "grouped" under the same *Resource*.
+This processor re-associates spans, log records and metric datapoints to a *Resource* that matches 
```

Comment:
Any reasons for converting this file from soft wraps to hard wraps?

Reply:
Ahh, my bad, will fix this in the upcoming update